modules/virt-troubleshoot-fusion-access-san.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/storage/install-configure-fusion-access-san.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshoot-fusion-access-san_{context}"]
+= Troubleshooting {IBMFusionFirst}
+
+If you encounter issues with {IBMFusionFirst}, provide the must-gather image to Red{nbsp}Hat support. This image contains critical data about your cluster and project resources, logs, and events from your deployment.
+
+.Procedure
+
+. To obtain the deployed version of {IBMFusionFirst}, use the following command:
++
+[source,terminal]
+----
+$ oc get fusionaccesses.fusion.storage.openshift.io  -n ibm-fusion-access fusionaccess-sample -o jsonpath='{.spec.storageScaleVersion}'
+----
++
+[NOTE]
+====
+This command returns the numeric value of the deployed version of {IBMFusionFirst} such as `2.11.0`.
+====
+
+. To create the must-gather image, use the following command:
```

Comment:
```suggestion
. To create the `must-gather` image, use the following command:
```
Funny enough, I went to check on this, and you can see in https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html/support/gathering-cluster-data#gathering-data-specific-features_gathering-cluster-data where it's formatted differently for Virtualization versus regular OCP 🤭 

<img width="848" height="502" alt="Image" src="https://github.com/user-attachments/assets/1d8d909c-7cd1-409d-a484-f17475755064" />

I think it _should_ be formatted as "`must-gather` image" but we probably need a follow up Jira to fix it in other places in the docs too.